packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -19,9 +19,35 @@ const meta: ComponentMeta<typeof IconButton> = {
       page: mdx,
     },
   },
+  args: {
+    name: 'apps',
+    size: 'md',
+    iconType: 'default',
+    tooltip: 'sample tooltip message',
+    tooltipPlacement: 'top',
+    variant: 'secondary',
+    ariaLabel: 'sample aria-label content',
+  },
+  argTypes: {
+    name: {
+      control: 'select',
+    },
+    size: {
+      control: 'select',
+    },
+    iconType: {
+      control: 'radio',
+    },
+    tooltipPlacement: {
+      control: 'select',
+    },
+    variant: {
+      control: 'radio',
+    },
```

Comment:
storybook should infer these automatically - i think the only one we really need is to set the tooltip control to text (because the type is `ReactNode | string`, so without explicitly setting the control it defaults to some generic input)
```suggestion
    tooltip: {
      control: 'text'
    },
```

Reply:
Makes sense, I was debating on whether or not to explicitly show them, but you are correct, if it is inferred automatically by storybook we should probably not be including them.